googlesource/source_intergration_test.go

```diff
@@ -37,6 +36,7 @@ var (
 	tableID          = "conduit_test_table"
 	tableIDTimeStamp = "conduit_test_table_time_stamp"
 	location         = "US"
+	globalCounter    = 0
 )
 
```

Comment:
There's a typo in the file name, source_inteRgration_test.

Reply:
done